src/UnityNuGet.Tests/RegistryTests.cs

```diff
@@ -140,7 +140,9 @@ public async Task Ensure_Min_Version_Is_Correct_Ignoring_Analyzers_And_Native_Li
                 @"Dapplo.Windows.Messages",
                 @"Dapplo.Windows.User32",
                 // It has too many versions, the minimum version is lifted so as not to process so many versions
-                @"UnitsNet.*"
+                @"UnitsNet.*",
+                // Most versions < 1.7.0 don't target .netstandard2.0
+                @"XLParser"
```

Comment:
Why is it necessary to ignore it? I see that 1.7.0 is the first one that supports .netstandard 2.0, isn't it? And it is the one you have put in registry.json, so you should not need to ignore it in the test, right?

Reply:
I was hosting the server locally via docker and I tried the lowest version possible (1.1.3). It seems that a couple of the first versions can be converted for upm but then no other version supports .netstandard 2.0 until 1.7.0.
I didn't like the idea of fragmenting the supported versions but I was afraid that the ci / build check would fail on the minimum version test too; so I decided to put it in the exclusions for the test.

Please let me know If you would like to try without the exclusion, worst scenario is that we have to wait for two ci / build jobs.

Reply:
Ehy! You can find the log of my docker execution attached. I just have run "docker compose up -d" with the example docker-compose.yml, providing a registry.json that contained only two entries: Irony and XLParser (v1.1.3).

[docker.log](https://github.com/xoofx/UnityNuGet/files/14635040/docker.log)